content/en/continuous_integration/setup_tests/dotnet.md

```diff
@@ -71,38 +53,28 @@ dd-trace --help
 
 The following list shows the default values for key configuration settings:
 
-
-`--set-ci`
-: Sets up the clr profiler environment variables for all the CI pipeline. <br/>
-**Default**: `false`
-
-`--dd-env`
-: Environment name for unified service tagging.<br/>
-**Environment variable**: `DD_ENV`<br/>
-**Default**: `(empty)`
-
 `--dd-service`
-: Service name for unified service tagging. <br/>
+: Name of the service or library under test.<br/>
 **Environment variable**: `DD_SERVICE`<br/>
-**Default**: `(ProcessName)` 
+**Default**: `(ProcessName)`<br/>
+**Example**: `my-dotnet-app`
 
-`--dd-version` 
-: Version for unified service tagging. <br/>
-**Environment variable**: `DD_VERSION`<br/>
-**Default**: `(empty)`
+`--dd-env`
+: Name of the environment where tests are being run.<br/>
+**Environment variable**: `DD_ENV`<br/>
+**Default**: `(empty)`<br/>
+**Examples**: `local`, `ci`
 
-`--agent-url` 
-: Datadog trace agent URL. <br/>
+`--agent-url`
+: Datadog Agent URL for traces.<br/>
 **Environment variable**: `DD_TRACE_AGENT_URL`<br/>
 **Default**: `http://localhost:8126`
 
-Additionally, all [Datadog Tracer configuration][2] options can be used during test phase. 
-
-For example, to run a test suite with a custom agent URL and a custom service name:
+`--set-ci`
+: Sets up the clr profiler environment variables for all the CI pipeline. <br/>
+**Default**: `false`
```

Comment:
@tonyredondo do we need this here?

Reply:
Not for now, or at least not in this way, could mislead the user. 

I'll work to increase the CI support for that option (only Azure Pipeline is supported) first.

Reply:
Ok, I'll remove it for now and we can readd it later if needed